k8s/src/main/scala/io/buoyant/k8s/Stabilize.scala

```diff
@@ -0,0 +1,60 @@
+package io.buoyant.k8s
+
+import com.twitter.util.{Activity, Updatable, Var}
+
+trait Stabilize {
```

Comment:
I'm not a huge fan of providing utility methods as mixins like this.  I'd prefer to put these utility methods in a static `object` or on the package object.